package.json

```diff
@@ -37,7 +37,8 @@
     "recovery-test": "yarn workspace recovery-test",
     "ts-integration": "yarn workspace ts-integration",
     "utils": "yarn workspace utils",
-    "zk": "yarn workspace zk"
+    "fmt:check": "prettier --check \"**.{js,jsx,ts,tsx,vue,md}\" --cache",
+    "fmt": "prettier --write \"**.{js,jsx,ts,tsx,vue,md}\" --cache"
```

Comment:
why is this needed? should we not use zkstack dev fmt? Or that triggers this? But now it would have been here before

Reply:
Yeah, it's part of the new formatter, i was forced to merge